test/io/serializers/arrays.default.good

```diff
@@ -363,11 +363,11 @@ SUCCESS: [domain(4,int(64),one)] int(64)
 SUCCESS: [domain(5,int(64),one)] int(64)
 --- array of arrays ---
 ===== writing: =====
-1 2 3 2 4 6 3 6 9
+1.11235 1.21235 1.31235 2.11235 2.21235 2.31235 3.11235 3.21235 3.31235
 --------------------
-1 2 3 2 4 6 3 6 9
+1.11235 1.21235 1.31235 2.11235 2.21235 2.31235 3.11235 3.21235 3.31235
 ====================
-SUCCESS: [domain(1,int(64),one)] [domain(1,int(64),one)] int(64)
+SUCCESS: [domain(1,int(64),one)] [domain(1,int(64),one)] real(64)
```

Comment:
But I like the idea of checking the real array comparisons.  Maybe also do that version without printing the contents of the array?

Reply:
Or alternatively have a prediff that will accept variation on the values for that line?